_posts/2021-01-12-github-application-running-on-faasd-behind-inlets.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: How to expose your Github application running locally on faasd to the internet ?
+description: In this guide, we are going to demonstrate how we can build our Github Application and run it locally on our faasd instance then expose it to the internet using inlets to be able to Github can send events to it
+tags: inlets-pro inletsctl go github-application faasd rasperry-pi caddy tls
+author_staff_member: developer-guy
+dark_background: true
+image: /images/2021-01-12-github-application-using-go-and-inlets-pro/faasd-issue-bot.png
+date: 2021-01-12
+---
+
+## Introduction
+
+In this guide, we are going to develop a [Github application](https://docs.github.com/en/free-pro-team@latest/developers/apps) using Go, then we deploy it as a serverless function to make use of [faasd](https://github.com/openfaas/faasd) which is a lightweight & portable faas engine.We are also going to do this demo on our local environment, so we should open our function which runs on our local environment to the Internet so Github can send events to our function.In order to do that we use inlets-pro which provides secure TCP/L4 tunnels.
```

Comment:
Change to: 

In this guide, we are going to develop a "GitHub App" using Go.

Then what I'd like to see is you talk about what a GitHub App is, the use-case, why they are better than repo webhooks, and also give two examples: derek and openfaas cloud and explain briefly how they use them.

We know, but the reader likely will not.

Then after that, feel free to keep the technical overview.

Space missing here "function.In"

Reply:
done